lib/check-base-branch.js

```diff
@@ -36,10 +36,24 @@ const getPRNumber = () => {
         return pr.number;
     }
 };
-const getProtectedBranches = () => {
-    const branches = core.getInput("protected-branches", { required: true });
+const getBranches = (key, isRequired) => {
+    const branches = core.getInput(key, { required: isRequired });
     return branches.split(',').map(b => b.trim());
 };
+const getProtectedBranches = () => {
+    return getBranches("protected-branches", true);
+};
+const getExceptionBranches = () => {
+    return getBranches("exception-branches", false);
+};
+const getExceptionPrefixes = () => {
+    return getBranches("exception-prefixes", false);
+};
+const pullRequestIsExempt = (prBranch) => {
+    core.debug(`PR Branch: '${prBranch}'.`);
+    return getExceptionBranches().some(exceptionBranch => exceptionBranch === prBranch)
```

Comment:
Wanted to call out a bug with this logic. In the case that exception-prefixes is a blank string, getExceptionPrefixes will return `[""]` and the `prBranch.startsWith(prefix)` will return `true` for empty string. Thus, if no exception-prefixes are provided, the action always passes with true.

Reply:
Good catch. @danielpuent3 can you incorporate this feedback into your revision?